clients/fides-js/src/fides.ts

```diff
@@ -216,16 +217,20 @@ const init = async ({
     _Fides.geolocation = geolocation;
     _Fides.options = options;
     _Fides.initialized = true;
-    if (experience) {
+    if (experienceHasNotices(experience)) {
```

Comment:
is this basically to check if experience is not `{}`? if so, to keep this compatible with TCF, can we check that it's not `{}` (and that it is defined)? (TCF won't have notices!). I guess you can't really do `{} === {}` since those are technically different objects, but you could check if [it's an empty object](https://stackoverflow.com/questions/679915/how-do-i-test-for-an-empty-javascript-object)

if you're feeling _really_ fancy 🎩 you could use a TS predicate which might look something like...

```typescript
type EmptyExperience = {};
const isEmptyExperience = (obj: unknown): obj is EmptyExperience =>
  typeof obj === "object" && obj != null && Object.keys(obj).length === 0;
```

if you do some sort of check and change the TS type in the comment above, I don't think you'd need the cast to `PrivacyExperience` below

Reply:
oh thank you, I'd forgotten about TCF not using `notices`. We really have to get the TCF branch is `main` soon...